### ISSUE TITLE
Unmaintained?

### ISSUE THREAD

**Issue by wpdonders** [AmirPupko/pandas-to-sql#1]
Hi there,

I'm interested in the concept of writing pandas-like API to generate SQL to talk to relational databases. I was wondering if this project is current no longer maintained. 

Thanks!

**Comment by arne-rustad** [comment]
Yeah, very interesting concept. Really hope it is still maintained
